src/ble.ts

```diff
@@ -179,7 +179,7 @@ export class ImprovBluetoothLE extends EventTarget {
     const state = encodedState.getUint8(0) as ImprovCurrentState;
     this.logger.debug("improv current state", state);
     this.currentState = state;
-    this.dispatchEvent(new CustomEvent("state-change"));
+    this.dispatchEvent(new CustomEvent("state-changed"));
```

Comment:
We shouldn't fix the typo as that's a breaking change. It's been there and so that's it. Instead we should fix the readme.

Reply:
Hi, I think this is not just a typo. The modal dialog ('Press the authorize button on the device') waits endless after the charackteristic state is changed (e.g. state '1' to '2').
The dialog is advised to a "state-changed") but this event will never raised.

Reply:
Then let's fix the event listener. That way we can avoid a breaking change.